repl/plugins/filename_completion.lua

```diff
@@ -0,0 +1,63 @@
+local utils = require 'repl.utils'
+local lfs = require 'lfs'
+assert(lfs, 'LuaFileSystem is missing.')
```

Comment:
You don't need an `assert` here; `require` will fail if it can't find the module